<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MovieBar</class>
    <widget class="QWidget" name="MovieBar">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>501</width>
                <height>42</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
                <number>6</number>
            </property>
            <property name="topMargin">
                <number>6</number>
            </property>
            <property name="rightMargin">
                <number>6</number>
            </property>
            <property name="bottomMargin">
                <number>6</number>
            </property>
            <item>
                <widget class="QToolButton" name="prevBtn">
                    <property name="icon">
                        <iconset theme="previous">
                            <normaloff>.</normaloff>.
                        </iconset>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QToolButton" name="playBtn">
                    <property name="text">
                        <string>...</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QToolButton" name="nextBtn">
                    <property name="icon">
                        <iconset theme="next">
                            <normaloff>.</normaloff>.
                        </iconset>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QSlider" name="movieSlider">
                    <property name="pageStep">
                        <number>1</number>
                    </property>
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDoubleSpinBox" name="speedBox">
                    <property name="toolTip">
                        <string>Slow factor - time between each frame will me multiplied by this value</string>
                    </property>
                    <property name="buttonSymbols">
                        <enum>QAbstractSpinBox::PlusMinus</enum>
                    </property>
                    <property name="decimals">
                        <number>3</number>
                    </property>
                    <property name="minimum">
                        <double>0.001000000000000</double>
                    </property>
                    <property name="maximum">
                        <double>10.000000000000000</double>
                    </property>
                    <property name="singleStep">
                        <double>0.100000000000000</double>
                    </property>
                    <property name="value">
                        <double>1.000000000000000</double>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QToolButton" name="sweepBtn">
                    <property name="text">
                        <string>...</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QToolButton" name="sameWindowBtn">
                    <property name="toolTip">
                        <string>Try using same window during movie</string>
                    </property>
                    <property name="text">
                        <string>...</string>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QToolButton" name="sameTransformBtn">
                    <property name="toolTip">
                        <string>Try using same rotate/scale during movie</string>
                    </property>
                    <property name="text">
                        <string>...</string>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
